### PR TITLE
Fix teleporting to the calling player's bed

### PR DESCRIPTION
**Pull request by akrantz01**
When using the bed destination, the player's name would always be set causing the `b:playerbed` to not work. This adds an extra check to see if the player's name is `playerbed`. I also added the ability to use `b:my-bed` since it is an invalid player name and will never collide with an actual player' username (assuming Mojang/Microsoft doesn't change the username requirements).

### CODE REVIEW

**Comment by nicegamer7** [comment]
Thank you for taking the time to look into this issue. I'll give this PR a review sooner or later.

File: src/main/java/com/onarandombox/MultiverseCore/destination/BedDestination.java
Comment (benwoo1110) [inline src/main/java/com/onarandombox/MultiverseCore/destination/BedDestination.java:54]: I think we can just decide on one identifier to used as for the caller bed string. I think `playerbed` is ok already, since I googled on mcuuid and there is no such account name.

Comment (akrantz01) [reply src/main/java/com/onarandombox/MultiverseCore/destination/BedDestination.java:54]: Fair enough. I had just changed it to ensure that there would never be any conflicts, but its probably pretty unlikely that anyone would register and use the name `playerbed`.

Comment (nicegamer7) [reply src/main/java/com/onarandombox/MultiverseCore/destination/BedDestination.java]: I kinda disagree. I think we should change it so that there is no possibility for conflict. @dumptruckman any opinion on this? I guess it doesn't really matter... the username `playerbed` is kinda useless.

Comment (nicegamer7) [reply src/main/java/com/onarandombox/MultiverseCore/destination/BedDestination.java]: Actually, I think that would be a change for MV 5.

**Review by nicegamer7 — DISMISSED** [review]
LGTM 👍.

File: src/main/java/com/onarandombox/MultiverseCore/destination/BedDestination.java
Comment (benwoo1110) [inline src/main/java/com/onarandombox/MultiverseCore/destination/BedDestination.java:26]: `OLD_BED_STRING` doesnt sound like a good variable name to me. Something like `OWN_BED_STRING` would be preferred.

**Review by benwoo1110 — CHANGES_REQUESTED** [review]

**Review by nicegamer7 — APPROVED** [review]

**Comment by nicegamer7** [comment]
@akrantz01 I just want to ask if you've tested these changes? The changes look good to me, just want to make sure.

**Comment by akrantz01** [comment]
The original changes I tested on 1.16, 1.15, and 1.14, but the most recent ones I haven't been able to test since I'm on the road.

**Comment by nicegamer7** [comment]
Okay, I just tested it and it's all good 👍.

**Review by benwoo1110 — APPROVED** [review]

**Comment by nicegamer7** [comment]
Just so you're aware of what's going on @akrantz01, ben and I are waiting on approval from the main dev.

**Comment by benwoo1110** [comment]
Looks good to me, I think I got the greenlight (ish) to merge small bug fixes, so let's merge this 😃

**Pull request merged**
